cookbook/mass_edition/register_a_new_mass_edit_action.rst

```diff
@@ -57,9 +33,9 @@ The method ``getJobInstanceCode()`` is very important as it determines which job
 
 Once the Operation is created, you must register it as a service in the DIC with the ``pim_catalog.mass_edit_action`` tag:
```

Comment:
Sometime, it is "in the DIC", and some time "in the DI" choose one of them everywhere :).